systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.rbac.KubernetesClusterRoleBinding;
+import io.strimzi.test.annotations.ClusterOperator;
+import io.strimzi.test.annotations.Namespace;
+import io.strimzi.test.extensions.StrimziExtension;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.List;
+
+import static io.strimzi.test.extensions.StrimziExtension.REGRESSION;
+
+@ExtendWith(StrimziExtension.class)
+@Namespace(MultipleNamespaceST.CO_NAMESPACE)
+@Namespace(value = MultipleNamespaceST.SECOND_NAMESPACE, use = false)
+@ClusterOperator
+public class AllNamespaceST extends AbstractNamespaceST {
```

Comment:
From my POV we should have tests which deploy cluster components to more than one namespace, just for sure. Currently the tests are completely same as for watching 2 namespaces. For example create 2/3 additional namespaces, put CO to one, kafkas and TO to different namesapces... WDYT?

Reply:
The problem here, that we need to have resources on the virtual machine to deploy Kafka on three namespaces. Are sure that we have it? Are you sure that it will be a stable test? or maybe checking it once manually will be enough?

Reply:
Well you can try it :) but for example 3 namesapces - one for CO, one for kafka-source, one for kafka-target and put there mm should be fine :)

Reply:
Okay, it sounds reasonable and I updated ST to deploy 2 Kafka clusters in a different namespace than CO. So Now we have `CO` in namespace `co-namespace-test`, Kafka cluster in the namespace `second-namespace-test` and Kafka cluster in namespace `third-namespace-test`